templates/website/index-index.html

```diff
@@ -6,6 +6,10 @@
 
 <div class="row row-full-width hero">
     <div class="large-10 large-centered columns">
+        <a href="https://www.mysociety.org/about/anniversary" id="mysociety-badge" aria-label="Visit mySociety 20 years anniversary">
+            <img class="badge-desktop" src="/static/img/badge-mysociety-desktop@2x.png" width="60" height="80" alt="mySociety 20 Years badge">
```

Comment:
Minor thing, but if it were me, I’d use an alt text like "20 years of mySociety" (or whatever the text _actually inside the image_ is) for these two images.